src/main/java/org/jabref/logic/importer/util/ShortDOIServiceResponse.java

```diff
@@ -0,0 +1,54 @@
+package org.jabref.logic.importer.util;
+
+import java.util.Objects;
+
+public class ShortDOIServiceResponse {
```

Comment:
This class seems to be unused?!

Reply:
Oh yes... Sorry for that. I changed the approach during work and forgot about it. Removed it.